### PR TITLE
Add parse_str, rasterize_to_raw_rgba, and optional image.

### PR DESCRIPTION
**Pull request by tilde35**
Thanks for creating this library! I added a few features that I needed for my application usage. Feel free to use or modify as you see fit.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -189,14 +228,26 @@ impl Drop for SvgImage {
  * Loads SVG data from a file at the given `Path`.
  *
  * # Arguments
- * - `svg_path` - Path to the SVG you want to load
+ * - `svg_path` - Text contents of the SVG you want to load
```
Comment (nickbrowne) [inline src/lib.rs]: I think these comments got switched around by accident. Shouldn't this remain as "Path to the SVG you want to load"?

Then the documentation for `parse_str` seems like it should be "Text contents of the SVG you want to load".

**Comment by nickbrowne** [comment]
The changes you're proposing look good to me, I'll find them useful as well!

I've just got the one question about the comments for the function arguments, then I'd be happy to merge this.

File: src/lib.rs (hunk 1 above)
Comment (tilde35) [reply src/lib.rs]: Yep, looks like I copy-pasted at the wrong location. I'll get that updated. Thanks!

**Comment by tilde35** [comment]
All set, the comments have been updated. Thanks!

**Comment by nickbrowne** [comment]
Thanks, I'll try to publish a new version later today :+1:

**Pull request merged**